test/common/crypto/verify_signature_fuzz_test.cc

```diff
@@ -0,0 +1,31 @@
+#include "common/common/hex.h"
+#include "common/crypto/utility.h"
+
+#include "test/common/crypto/verify_signature_fuzz.pb.validate.h"
+#include "test/fuzz/fuzz_runner.h"
+
+namespace Envoy {
+namespace Common {
+namespace Crypto {
+namespace {
+
+DEFINE_PROTO_FUZZER(const test::common::crypto::VerifySignatureFuzzTestCase& input) {
+  auto const& key = input.key();
```

Comment:
Nit: I would have usually expected the pattern of `const auto&`, but I guess this works too. Most of Envoy is doing `const auto&` here.

Reply:
> Nit: I would have usually expected the pattern of `const auto&`, but I guess this works too. Most of Envoy is doing `const auto&` here.

Thanks! Fixed!